packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -242,7 +244,8 @@ function TableColumnHeader({column, state}) {
       style={{
         textAlign: column.colspan > 1 ? 'center' : 'left',
         padding: '5px 10px',
-        outline: isFocusVisible ? '2px solid orange' : 'none',
+        outline: 'none',
+        boxShadow: isFocusVisible ? 'inset 0 0 0 2px orange' : 'none',
```

Comment:
outline wasn't looking great in the resize example since it wasn't inset, replaced it with `boxShadow`. Did so for these pre-existing examples as well to reduce the diff between the this and the column resize examples